code/src/main/kotlin/com/expediagroup/sdk/core/request/initializer/InitializerFunctions.kt

```diff
@@ -11,7 +11,7 @@ object InitializerFunctions {
     }
 
     val attachDefaultRequestLoggingInterceptor = fun (request: HttpRequest) {
-        request.interceptor = HttpRequestLoggingInterceptor()
+        request.interceptor = com.expediagroup.sdk.core.request.interceptor.HttpRequestLoggingInterceptor()
```

Comment:
Same comment for the fully qualified name. (Applicable to related places).

Reply:
Done.